### PR TITLE
Dont start processes in init

### PR DESCRIPTION
**Pull request by maringuu**
This is needed for upcoming #895

### CODE REVIEW

**Comment by maringuu** [comment]
I spend a lot of time figuring out what caused the ci fail.
The result is [ca8af4d](https://github.com/fkie-cad/FACT_core/pull/914/commits/ca8af4d403c608c239c54e0b3f3f96e073ecd61a).
This sadly only is a description of the problem and a workaround.

Despite this the PR is ready for review.

@jstucke Any idea what causes this?

**Comment by maringuu** [comment]
Successful CI run: https://github.com/fkie-cad/FACT_core/actions/runs/3786408433

**Review by jstucke — APPROVED** [review]
Looks good apart from the "FIXME"

File: src/conftest.py [hunk 1]
```diff
@@ -211,30 +211,23 @@ def my_fancy_plugin(analysis_plugin)
                 start_processes=False,
             )
             def my_fancy_test(analysis_plugin, monkeypatch):
-                # Undo the patching of MyFancyPlugin.start_worker
-                monkeypatch.undo()
                 analysis_plugin.TIMEOUT = 0
-                # Now start the worker
-                analysis_plugin.start_worker()
+                # Now start the worker manually
```
[the remark this answers is not in the document]
Comment (maringuu) [reply src/conftest.py]: I left out the word completly.

**Pull request merged**